front/components/wrappers/AppointmentDataContext.tsx

```diff
@@ -0,0 +1,16 @@
+import { SetStateAction, createContext, useState, Dispatch } from 'react';
+import { WrapperType } from 'types/wrapper';
+
+const AppointmentDataContext = createContext<any>({} as any);
```

Comment:
dont you know what type do you want to store inside context?

Reply:
I know, but I have a problem with execute it. Esspecially with useState "setSth" function (in this case setAppointmentID)

Reply:
You can put it as TechnicalExcelence or write some TODO. Maybe at the end of the project you will get some ideas